temp.py

```diff
@@ -0,0 +1,2 @@
+import pdb
+print("hello")
```

Comment:
**CODE_QUALITY**: Instead of using `print`, consider using the `logging` module to provide more control over logging levels and outputs.
```
import logging

logging.basicConfig(level=logging.INFO)
logging.info('hello')
```